build-info-extractor/src/main/java/org/jfrog/build/extractor/clientConfiguration/util/AqlUtils.java

```diff
@@ -0,0 +1,189 @@
+package org.jfrog.build.extractor.clientConfiguration.util;
+
+import org.apache.commons.lang.ArrayUtils;
+import org.apache.commons.lang.StringUtils;
+import org.jfrog.build.api.search.AqlSearchResult;
+import org.jfrog.build.api.util.Log;
+import org.jfrog.build.extractor.clientConfiguration.client.ArtifactoryDependenciesClient;
+
+import java.io.IOException;
+import java.util.*;
+
+public class AqlUtils {
```

Comment:
Can we move all the code of this class into the *AqlHelperBase* and make it protected?